cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst

```diff
@@ -110,7 +110,7 @@ See `config instructions <https://github.com/openedx/i18n-tools#transifex-comman
 transifex credentials.
 
 See `transifex documentation <https://docs.transifex.com/integrations/django>`_ for more details about integrating
-django with transiflex.
+django with transifex.
```

Comment:
Please link to https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html instead.